source/meetings/2018/september/index.html.md

```diff
@@ -0,0 +1,96 @@
+---
+updated_by:
+  email: tatiana.stantonian@lrug.org
+  name: Tatiana Stantonian
+created_by:
+  email: tatiana.stantonian@lrug.org
+  name: Tatiana Stantonian
+category: meeting
+title: September 2018 Meeting
+updated_at:
+published_at: 2018-08-24 13:00:00 +0000
+created_at: 2018-08-24 13:00:00 +0000
+parts: {}
+status: Published
+---
+
+The September 2018 meeting of LRUG will be on *Monday the 11th of September*,
```

Comment:
It's on Tuesday, not Monday.  The date is right though!

Reply:
Silly me. Thanks, amended now!

Reply:
I've done the same on countless occasions!